plugins/events/create.js

```diff
@@ -197,7 +197,8 @@ module.exports = () => ({
                                     .then((parentEvent) => {
                                         payload.baseBranch = parentEvent.baseBranch || null;
 
-                                        if (parentEvent.meta && parentEvent.meta.parameters) {
+                                        if ((!payload.meta || !payload.meta.parameters) &&
+                                            parentEvent.meta && parentEvent.meta.parameters) {
```

Comment:
on restart scenarios does `payload.meta.parameters` include entire parameters ? Do we need to do a `merge` ?

cc @adong

Reply:
yeah it includes entire parameters as long as there's change in default parameters.